### PR TITLE
Happy block: Support content global footer

### PR DESCRIPTION
**Pull request by escapemanuele**
NOTE: There will probably be some future iterations on this, as we wait for [this](p9Jlb4-6Vx-p2#comment-7533) to be answered

[Context](IceKRdz5qlPYwuqzu6pBgF-fi-1815_26190) - Point 6

This PR creates a new block in the Happy Blocks library, to be used by /support /forums and /learn.

image

## Testing
1. Checkout branch
2. `cd apps\happy-blocks`
3. `yarn dev --sync`

Related to https://github.com/Automattic/wpsupport3/pull/391

### CODE REVIEW

File: apps/happy-blocks/block-library/support-content-footer/index.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+				<input type="hidden" name="blog_id" value="<?php echo get_current_blog_id(); ?>">
+				<input type="hidden" name="sub-type" value="subscribe-block">
+				<input type="hidden" name="redirect_fragment" value="subscribe-blog">
+				<input type="hidden" id="_wpnonce" name="_wpnonce" value="cfa1e9a8e2">
+				<button class="support-content-subscribe-submit" type="submit">
+					<?php esc_html_e( 'Subscribe', 'happy-blocks' ); ?>
+				</button>
+			</form>
+			<p class="support-content-subscribe-disclaimer">
+				<?php esc_html_e( 'Your information will be used in accordance with WordPress.com privacy policy.', 'happy-blocks' ); ?>
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/index.php]: PP should be a link to https://automattic.com/privacy/

File: apps/happy-blocks/block-library/support-content-footer/style.scss [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+$breakpoint-mobile: 782px; //Mobile size.
+$breakpoint-tablet: 1224px; //Large tablet size.
+$breakpoint-desktop: 1440px; //Desktop size.
+$search-icon: 'data:image/svg+xml,%3Csvg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none"%3E%3Cpath d="M9.16667 15.8333C12.8486 15.8333 15.8333 12.8486 15.8333 9.16667C15.8333 5.48477 12.8486 2.5 9.16667 2.5C5.48477 2.5 2.5 5.48477 2.5 9.16667C2.5 12.8486 5.48477 15.8333 9.16667 15.8333Z" stroke="%238C8F94" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/%3E%3Cpath d="M17.5 17.5L13.875 13.875" stroke="%238C8F94" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/%3E%3C/svg%3E';
+$recoleta-font-family: "Recoleta", sans-serif;
+$system-font-family: -apple-system, BlinkMacSystemFont, "Noto Sans", "Segoe UI", "Roboto",
+	"Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
+
+.support-content-footer {
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/style.scss]: Can we wrap this whole thing with something that starts with `happy-blocks`?

File: apps/happy-blocks/block-library/support-content-footer/style.scss [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+				background-color: var(--color-primary);
+				border-radius: 0 !important;
+				border-width: 1px;
+				color: var(--color-primary-0);
+				font-size: 1rem;
+				line-height: 1.3;
+				margin: 0;
+				margin-left: 10px;
+				padding: 15px 23px;
+			}
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/style.scss]: Both of these need some border love (namely color and radius).

image

**Review by alshakero — CHANGES_REQUESTED** [review]
Great work. I love these for some reason. Left some tiny comments.

**Comment by escapemanuele** [comment]
Thank you @alshakero, I think I addressed all the feedback ✌️

File: apps/happy-blocks/block-library/support-content-footer/style.scss [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+				padding: 16px;
+				width: 100%;
+				height: 40px;
+			}
+
+			button.support-content-subscribe-submit {
+				background-color: var(--color-primary);
+				border-radius: 4px !important;
+				border-width: 1px;
+				color: var(--color-primary-0);
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/style.scss:171]: Needs a `border: none` and `cursor: pointer`. And maybe some opacity on hover.

File: apps/happy-blocks/block-library/support-content-footer/style.scss [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+		.support-content-cta-right {
+			flex: 1;
+
+			figure {
+				margin-bottom: 0;
+				height: 100%;
+
+				img {
+					width: 100%;
+					height: 100%;
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/style.scss]: This ruins the aspect ratio of the picture. One of them should be 100%, I think height.

File: apps/happy-blocks/block-library/support-content-footer/style.scss [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+		margin-left: auto !important;
+		margin-right: auto !important;
+		max-width: var(--wp--style--global--wide-size);
+	}
+
+	.support-content-cta {
+		border: 1px solid var(--studio-gray-5);
+		display: flex;
+		margin-bottom: 2rem;
+		font-size: 1.25rem;
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/style.scss]: Needs border-radius.

**Review by alshakero — CHANGES_REQUESTED** [review]
Super. Left some tiny CSS issues.

File: apps/happy-blocks/block-library/support-content-footer/index.php [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+				<?php esc_html_e( 'New to WordPress.com? ', 'happy-blocks' ); ?>
+				<a href="<?php echo esc_url( 'https://wordpress.com/support/start/' ); ?>" target="_blank" rel="noreferrer noopener">
+					<?php esc_html_e( 'Find your perfect-fit plan here.', 'happy-blocks' ); ?>
+				</a>
+			</p>
+		</div>
+		<div class="support-content-subscribe">
+			<p><?php esc_html_e( 'Sign up for educational resources updates:', 'happy-blocks' ); ?></p>
+			<form action="https://subscribe.wordpress.com" method="post" accept-charset="utf-8" data-blog="<?php echo get_current_blog_id(); ?>" data-post_access_level="everybody" id="subscribe-blog">
+				<input class="support-content-subscribe-email" required="required" type="email" name="email" placeholder="<?php esc_html_e( 'Enter your email address', 'full-site-editing' ); ?>"  id="subscribe-field">
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/index.php]: Incorrect text-domain. It's from the linter.

File: apps/happy-blocks/block-library/support-content-footer/index.php [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+			<h4 class="support-content-resource__title has-recoleta-font-family">
+				<?php esc_html_e( 'Support Guides', 'happy-blocks' ); ?>
+			</h4>
+			<p>
+				<?php esc_html_e( 'Find the answer to the questions you know you have about WordPress.com.', 'happy-blocks' ); ?>
+			</p>
+			<div class="resource-link">
+				<p>
+					<a href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/support/' ) ); ?>">
+						<?php esc_html_e( 'Search Guides', 'full-site-editing' ); ?>
```
Comment (alshakero) [inline apps/happy-blocks/block-library/support-content-footer/index.php]: +1

**Review by alshakero — APPROVED** [review]
Tiny important mistake. Then ship!

**Pull request merged**